pkg/apiserver/apiserver.go

```diff
@@ -76,7 +76,9 @@ var (
 	// to versioned API objects and back.
 	parameterCodec = runtime.NewParameterCodec(Scheme)
 	// #nosec G101: false positive triggered by variable name which includes "token"
-	TokenPath = "/var/run/antrea/apiserver/loopback-client-token"
+	TokenPath         = "/var/run/antrea/apiserver/loopback-client-token"
+	CertDir           = "/var/run/antrea/antrea-controller-tls"
```

Comment:
I don't think we need to access `CertDir` and `SelfSignedCertDir` from a file outside the current package, so no need to use a capital?

Reply:
It's used for unit-test.